Source/CarthageKit/Xcode.swift

```diff
@@ -254,7 +254,7 @@ public func schemesInProject(project: ProjectLocator) -> SignalProducer<String,
 		}
 		.skipWhile { line in !line.hasSuffix("Schemes:") }
 		.skip(1)
-		.takeWhile { line in !line.isEmpty }
+		.takeWhile { line in !line.isEmpty && line.hasPrefix("  ") }
```

Comment:
Agreed, though it seemed less fragile than any ideas I'd thought of to detect the 'noise' directly.